src/wp-admin/includes/meta-boxes.php

```diff
@@ -583,10 +583,28 @@ function post_tags_meta_box( $post, $box ) {
 		<label for="tax-input-<?php echo $tax_name; ?>"><?php echo $taxonomy->labels->add_or_remove_items; ?></label>
 		<p><textarea name="<?php echo "tax_input[$tax_name]"; ?>" rows="3" cols="20" class="the-tags" id="tax-input-<?php echo $tax_name; ?>" <?php disabled( ! $user_can_assign_terms ); ?> aria-describedby="new-tag-<?php echo $tax_name; ?>-desc"><?php echo str_replace( ',', $comma . ' ', $terms_to_edit ); // textarea_escaped by esc_attr() ?></textarea></p>
 	</div>
-	<?php if ( $user_can_assign_terms ) : ?>
+
+	<?php
+	if ( $user_can_assign_terms ) :
+		$tags = get_tags(
+			array(
+				'fields' => 'names',
```

Comment:
Adding this solve [my comment](https://github.com/ClassicPress/ClassicPress/pull/1287#issuecomment-1874038714).
`'hide_empty' => false,` 

Reply:
Now added